addons/l10n_in_pos/static/src/overrides/models/order.js

```diff
@@ -9,6 +9,95 @@ patch(Order.prototype, {
+            const taxes_ids = line.tax_ids || line.product.taxes_id;
+            if (hsnCode && taxes_ids) {
+                const product_taxes = line.pos.get_taxes_after_fp(taxes_ids, fiscalPosition);
+                const price_unit = line.get_unit_price();
+                const quantity = line.get_quantity()
+                const all_taxes = line.compute_all(product_taxes, price_unit, quantity, line.pos.currency.rounding);
+                //It is require to pre-compute GST Rate as we may have same hsn with different gst rates
+                const gstRate = this._computeL10nInGSTRate(all_taxes.taxes)
+                const groupKey = ''.concat(hsnCode, '-', gstRate)
+                if (!hsnSummary.hasOwnProperty(groupKey)){
+                    hsnSummary[groupKey] = {
+                        l10n_in_hsn_code: hsnCode,
+                        gst_rate: gstRate,
+                        SGST: {rate: 0, amount: 0},
+                        CGST: {rate: 0, amount: 0},
+                        IGST: {rate: 0, amount: 0},
+                        CESS: {amount: 0},
+                    }
+                }
+                const l10nInTaxKeyByTagRef = this._getl10nInTaxKeyByTagRef()
+                all_taxes.taxes.forEach((tax) => {
+                    tax.repartition_line_ids.forEach((line) => {
+                        line.tag_ids.forEach((tag) => {
+                            let tagRef = accountTagByXmlRefId[tag.id]
+                            let factor_percent = line.factor_percent
+                            const taxKey = l10nInTaxKeyByTagRef[tagRef]
+                            switch (taxKey) {
+                                case 'CGST':
+                                case 'SGST':
+                                case 'IGST':
+                                    hsnSummary[groupKey][taxKey]['amount'] += factor_percent * tax.amount / 100
+                                    hsnSummary[groupKey][taxKey]['rate'] = factor_percent * tax.tax_rate / 100
+                                    break;
+                                case 'CESS':
+                                    hsnSummary[groupKey][taxKey]['amount'] += factor_percent * tax.amount / 100
+                                    break;
+                            }
+                        })
+                    })
+                });
+            }
+        });
+        const hsnSummaryLines = Object.keys(hsnSummary)
+        const currency = this.pos.currency
+        hsnSummaryLines.forEach(line => {
+            const line_value = hsnSummary[line]
+            line_value.CGST.amount = this.env.utils.formatCurrency(line_value.CGST.amount, currency)
+            line_value.SGST.amount = this.env.utils.formatCurrency(line_value.SGST.amount, currency)
+            line_value.IGST.amount = this.env.utils.formatCurrency(line_value.IGST.amount, currency)
+            line_value.CESS.amount = this.env.utils.formatCurrency(line_value.CESS.amount, currency)
```

Comment:
```suggestion
            line_value.CGST.amount = this.env.utils.formatCurrency(line_value.CGST.amount)
            line_value.SGST.amount = this.env.utils.formatCurrency(line_value.SGST.amount)
            line_value.IGST.amount = this.env.utils.formatCurrency(line_value.IGST.amount)
            line_value.CESS.amount = this.env.utils.formatCurrency(line_value.CESS.amount)
```

the second parameter of the `formatCurrency` is a boolean asking whether you want to formatted value to contain the currency symbol or not. If you want the currency to be display, you don't need to pass anything. If want the currency symbol to be hidden, you should pass `false` as second argument.